ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -129,3 +147,25 @@ def send_metrics_from_properties(self, properties, metrics_map, prefix, tags):
                     )
                     return
                 self.send_metric(metric_type, metric_full_name, metric_value, new_tags)
+
+    def check_queue_manager_process(self):
+        if self._config.queue_manager_process_pattern is None:
+            return True
+
+        return self.process_matcher.check_condition(self.check_id, self._config.queue_manager_process_pattern, self.log)
+
+    def reset_queue_manager_process_match(self):
+        if self._config.queue_manager_process_pattern is not None:
+            self.log.debug('Resetting queue manager process match')
+            return self.process_matcher.remove(self.check_id)
+
+    def parse_config(self):
```

Comment:
```suggestion
    def create_process_matcher(self):
```
This is not parsing anything anymore